models/comments.models.js

```diff
@@ -0,0 +1,17 @@
+const db = require("../db/connection");
+exports.fetchCommentsByArticleId = (article_id) => {
+  return db
+    .query(
+      "SELECT * FROM comments WHERE article_id = $1 ORDER BY created_at DESC",
+      [article_id]
+    )
+    .then(({ rows }) => {
+      if (rows.length === 0) {
+        return Promise.reject({
+          status: 404,
+          message: "no comments found for this article id",
+        });
+      }
```

Comment:
You may need some more logic here to pass the new test that you will be adding in